pkg/cmd/pulumi/plugin_install.go

```diff
@@ -166,8 +151,8 @@ func newPluginInstallCmd() *cobra.Command {
 		"file", "f", "", "Install a plugin from a tarball file, instead of downloading it")
 	cmd.PersistentFlags().BoolVar(&reinstall,
 		"reinstall", false, "Reinstall a plugin even if it already exists")
-	cmd.PersistentFlags().BoolVar(&verbose,
-		"verbose", false, "Print detailed information about the installation steps")
+	//cmd.PersistentFlags().BoolVar(&verbose,
```

Comment:
Is this no longer needed?

Reply:
So this was overriding the verbosity that was command wide and what was actually causing the confusion